scripts/vcs_wrapper

```diff
@@ -15,6 +15,11 @@ for file in $files; do
   file_args+=( "$file" )
 done
 
+filelists="$WAKE_IP_RESOURCE_FILELIST"
+filelistargs=( )
```

Comment:
```suggestion
filelist_args=( )
```

For consistency with all of the other variables, can you put an underscore before the word "args" in this variable?

Reply:
I have tested in soc-sesame environment with the above changes, the build seems clean to me.